clang/test/lab1/kashirin_alexander/Rename_Plugin_test.cpp

```diff
@@ -0,0 +1,211 @@
+// RUN: split-file %s %t
+// RUN: %clang_cc1 -load %llvmshlibdir/rename_id_plugin%pluginext\
```

Comment:
Also as for the tests, please add checks for replacement for classes with templates, static methods, inheritance and polymorphism (virtual)

Reply:
replacement checks added